third_party/terraform/utils/config.go.erb

```diff
@@ -837,7 +836,6 @@ func ConfigureBasePaths(c *Config) {
 	c.DnsBetaBasePath = DnsBetaDefaultBasePath
 	c.IamCredentialsBasePath = IamCredentialsDefaultBasePath
 	c.ResourceManagerV2Beta1BasePath = ResourceManagerV2Beta1DefaultBasePath
-	c.IAMBasePath = IAMDefaultBasePath
```

Comment:
I removed this since this now gets auto-generated with the `products.map.each` in line 825